includes/functions-html.php

```diff
@@ -757,7 +760,7 @@ function yourls_html_menu() {
 	$admin_sublinks = yourls_apply_filter( 'admin_sublinks', $admin_sublinks );
 	
 	// Now output menu
-	echo '<ul id="admin_menu">'."\n";
+	echo '<nav role="navigation"><ul id="admin_menu">'."\n";
```

Comment:
Nop :-)
Because `nav` doesn't have the same HTML role definition as an `ul` so keep `<nav role="navigation"><ul id="admin_menu">`